ports/mimxrt/machine_uart.c

```diff
@@ -98,6 +108,32 @@ bool lpuart_set_iomux(int8_t uart) {
     }
 }
 
+bool lpuart_set_iomux_rts(int8_t uart) {
+    int index = (uart - 1) * 2;
```

Comment:
Might be a good idea to put a static assertion in this function to assert that the size of `iomux_table_uart` and `ioumux_table_uart_cts_rts` tables are the same.

Reply:
Done, also fixed a typo (/cst/cts/)